config/config.yaml

```diff
@@ -29,17 +29,12 @@ ref:
     # if active, reads will be aligned to given pangenome instead of to the linear reference genome
     # Important: this is only supported for homo_sapiens so far
     activate: true
-    # URL to pangenome haplotypes (vcf-file)
+    # URL to pangenome haplotypes (gbz- and hapl-indexes)
     # Graph resources v1.1: https://github.com/human-pangenomics/hpp_pangenome_resources/
     # Graph resources v1.0: https://github.com/human-pangenomics/hpp_pangenome_resources/blob/main/hprc-v1.0-mc.md
     # Important: ensure that the haplotype vcf is built against the same reference genome as specified above under build
-    vcf: https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/freeze/freeze1/minigraph-cactus/hprc-v1.1-mc-grch38/hprc-v1.1-mc-grch38.raw.vcf.gz
-    # optional: In case contigs of the hoplotype vcf need to be renamed
-    # a list of python expressions can be defined.
-    # Contigs can be accessed by `contig`.
-    # Each expression will be successively saved into `renamed_contig`
-    rename_expressions:
-      - "'MT' if contig == 'chrM' else contig[3:]"
+    hapl: https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/freeze/freeze1/minigraph-cactus/hprc-v1.1-mc-grch38/hprc-v1.1-mc-grch38.hapl
+    gbz: https://s3-us-west-2.amazonaws.com/human-pangenomics/pangenomes/freeze/freeze1/minigraph-cactus/hprc-v1.1-mc-grch38/hprc-v1.1-mc-grch38.gbz
```

Comment:
Why not doing this in the code, and just requiring an hprc release here?

Reply:
And checking that species is human and build is supported?
Would be more convenient for the user, and we can be sure that stuff is properly handled.